usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/sample/XWalkEmbeddedAPISample.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2014 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.xwalk.embedded.api.sample;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import android.app.ListActivity;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+public class XWalkEmbeddedAPISample extends ListActivity {
```

Comment:
Since XWalkEmbeddedAPISample is the first Activity, can we use this Activity to initialize XWalkView?

If so, we can remove the XWalkInitializer from other activities.